web/src/compose_validate.ts

```diff
@@ -640,7 +680,7 @@ function validate_stream_message(scheduling_message) {
 
     const sub = stream_data.get_sub_by_id(stream_id);
     if (!sub) {
-        return validation_error("does-not-exist", stream_id);
+        return validation_error("does-not-exist", stream_name);
```

Comment:
This should pass `stream_id.toString()` instead of doing this `stream_name` lookup that will fail -- the whole point is that we're telling the user that a stream indicated by an ID does not exist.